spec.emu

```diff
@@ -15,14 +15,16 @@ contributors:
   - Rick Waldron
 </pre>
 
-<emu-clause id="sec-hasidentity" aoid="HasIdentity">
-  <h1>HasIdentity ( _argument_ )</h1>
-  <p>The abstract operation HasIdentity takes argument _argument_ (an ECMAScript language value). It determines if _argument_ is considered to have a unique identity, enabling _argument_ to be a key of a WeakMap or entry in a WeakSet. This operation performs the following steps when called:</p>
+<emu-clause id="sec-isobjectorsymbol" aoid="IsObjectOrSymbol">
+  <h1>IsObjectOrSymbol ( _argument_ )</h1>
+  <p>The abstract operation IsObjectOrSymbol takes argument _argument_ (an ECMAScript language value) and determines if its Type is Object or Symbol. This operation performs the following steps when called:</p>
```

Comment:
If we want to match styles for consistency, here are some options:

[IsArray](https://tc39.es/ecma262/#sec-isarray), [IsRegExp](https://tc39.es/ecma262/#sec-isregexp):

> The abstract operation IsObjectOrSymbol takes argument _argument_. It performs the following steps when called:

[IsCallable](https://tc39.es/ecma262/#sec-iscallable):

> The abstract operation IsObjectOrSymbol takes argument _argument_ (an ECMAScript language value). It determines if _argument_ is an object or a symbol. It performs the following steps when called:

---

There are inconsistencies in ECMA-262 using "an Object" or "an object", I honestly just prefer what is better accepted by the editors.